README.md

```diff
@@ -82,7 +82,7 @@ ASK CLI will create the skill and the lambda function for you. The Lambda functi
 8. Click on **New Application** on top right. Now we need to give it an *Application Name* and a *Redirect URI*. 
  
 9. For *Application Name* use **"alexa"**. This can be anything else as well. And for the *Redirect URI*, go back to Amazon Developer Console Account Linking page and at the bottom of the page you'll find some redirect URLs.
-Copy **https://pitangui.amazon.com/api/skill/link/YOURVENDORID** and paste it in the *Redirect URI* field. Click on save changes.
+Copy **https://pitangui.amazon.com/api/skill/link/YOURVENDORID** or **https://layla.amazon.com/api/skill/link/YOURVENDORID** and paste it in the *Redirect URI* field. Click on save changes.
```

Comment:
Thanks for mentioning this. Layla didn't work for me. I think it works based on where you have registered the dev account.
We should add the third one as well to this list.